bodhi/tests/server/test___init__.py

```diff
@@ -43,7 +43,8 @@ def test_exception(self, exception):
 
         self.assertIs(response, request_response)
         exception.assert_called_once_with(
-            "Unhandled exception raised:  OSError('Your money is gone.',)")
+            "Unhandled exception raised:  "
+            "OSError('Your money is gone.'{})".format(base._erepr_end))
```

Comment:
This solution is fine with me, but another way we could have done it is:

```
"Unhandled exception raised:  {}".format(OSError('Your money is gone.'))
```

This way we let the Python running the test serialize it as a string too, and it should serialize it consistently.

Reply:
Right. Will amend. However the second one might not be that simple.

Reply:
format doesn use repr, we need to use repr explicitly

Reply:
Ah, didn't realize that. Works for me.

Reply:
```pycon
>>> "Unhandled exception raised:  {}".format(OSError('Your money is gone.'))
'Unhandled exception raised:  Your money is gone.'
```